pkg/minikube/audit/audit.go

```diff
@@ -51,14 +54,40 @@ func args() string {
 }
 
 // Log details about the executed command.
-func Log(startTime time.Time) {
+func LogCommandStart() (string, error) {
 	if len(os.Args) < 2 || !shouldLog() {
-		return
+		return "", fmt.Errorf("this command should not be logged and len(os.Args) : %v, should be less than 2", len(os.Args))
```

Comment:
```suggestion
		return "", nil
```
We shouldn't be logging an error here